internal/subshell/subshell_test.go

```diff
@@ -19,10 +19,11 @@ import (
 	"github.com/ActiveState/cli/pkg/projectfile"
 )
 
-func setup(t *testing.T) {
+func setup(t *testing.T) { //nolint:unused
```

Comment:
I'm curious why this tag is needed. Is it because `setup()` is only called for some platform-specific tests?

Reply:
Yeah, it's only called in the windows tests so I've moved it there.